x-pack/plugins/uptime/public/components/queries/ping_list/get_pings.ts

```diff
@@ -23,26 +23,29 @@ export const getPingsQuery = gql`
       sort: $sort
       size: $size
     ) {
-      timestamp
-      http {
-        response {
-          status_code
+      Total
```

Comment:
Sorry, I should've noticed this before as well 😞 
```suggestion
      total
```